src/__e2e__/FileInputFieldWithList.stories.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+import { FileInputFieldWithList } from '../index.js'
+
+const singleFileArray = [new File([], 'dummy.txt')]
+
+const multipleFileArray = [
+    new File([], 'test1.md'),
+    new File([], 'duplicate.md', {
+        lastModified: 1581348175357,
+        type: 'text/markdown',
+    }),
+    new File([], 'test2.md'),
+]
```

Comment:
I see now why you've duplicated the `lastModified`. You could attach this one to the `window` object to have access in cypress.

Reply:
Good suggestion 👍 I've attached `duplicateFileConstructorArgs` to the window. This means the pre-populated File instance is different from the new File instance, but the files are constructed with the same arguments, so the reference file appears the same. This as close as we can get to reality IMO.